src/schemapack/spec/schemapack.py

```diff
@@ -41,7 +41,7 @@
     read_json_or_yaml,
 )
 
-SupportedSchemaPackVersions = Literal["0.1.0"]
+SupportedSchemaPackVersions = Literal["0.2.0"]
 SUPPORTED_SCHEMA_PACK_VERSIONS = typing.get_args(SupportedSchemaPackVersions)
```

Comment:
```suggestion
SUPPORTED_SCHEMA_PACK_VERSIONS: Final[list[str]] = ["0.2.0"]
```
Is this really a use case for `Literal`? Shouldn't you be using `Final` here without the intermediate object?

Reply:
In addition to the list of versions, we also need a type for pydantic. Do you see a possibility to realize that with your solution. Happy to discuss alternative solutions in a different PR.